Controller/ContentManagement/NatureController.php

```diff
@@ -72,21 +87,21 @@ public function reorderAction(ContentType $contentType, Request $request)
                     $revision->setRawData($data);
                     $dataService->finalizeDraft($revision);
                 } catch (\Exception $e) {
-                    $this->addFlash('warning', 'It was impossible to update the item '.$itemKey.': '.$e->getMessage());
+                    $this->addFlash('warning', 'It was impossible to update the item ' . $itemKey . ': ' . $e->getMessage());
                 }
             }
 
-            $this->addFlash('notice', 'The '.$contentType->getPluralName().' have been reordered');
-            
+            $this->addFlash('notice', 'The ' . $contentType->getPluralName() . ' have been reordered');
+
             return $this->redirectToRoute('data.draft_in_progress', [
-                    'contentTypeId' => $contentType->getId(),
+                'contentTypeId' => $contentType->getId(),
             ]);
         }
 
         return $this->render('@EMSCore/nature/reorder.html.twig', [
-                'contentType' => $contentType,
-                'form' => $form->createView(),
-                'result' => $result,
-         ]);
+            'contentType' => $contentType,
+            'form' => $form->createView(),
+            'result' => $result,
+        ]);
     }
 }
```

Comment:
tip: use different commits for logic changes & phpcs changes. Makes reviewing easier (and more complete)